CONTRIBUTING.md

```diff
@@ -17,26 +84,222 @@ augroup gno_autocmd
+the requested changes.
+
+If the PR _creator_ pushed code as a result of some conversation, they should link the commit hash in the relevant
+comment reply, ex:
+
+```Markdown
+Fixed in:
+
+[0a0577c](https://github.com/gnolang/gno/commit/0a0577ccdeb951a6621d6fbe1c04ac4e64a529c1)
+```
+
+#### Base PR template
+
+The [base PR template](https://github.com/gnolang/gno/blob/master/.github/pull_request_template.md) is by default a
+simple template meant to illustrate quickly what’s the context of the PR. Here, you should describe your PR in detail,
+and leave a remark as to how your changes have been tested.
+
+If you've run a manual test, please provide the exact steps taken.
+
+#### Detailed PR template
+
+The [detailed PR template](https://github.com/gnolang/gno/blob/master/.github/PULL_REQUEST_TEMPLATE/detailed_pr_template.md)
+is used when you want to convey a bit more detail and context for your PRs. In contrast to the default PR template, it
+follows a checklist approach that is meant to be filled out by the PR creator, with as much information as possible.
+These detailed PR descriptions serve an important purpose down the line in the future, when it’s important to understand
+the context under which a PR has been created, and for what reason.
+
+You can utilize the detailed PR template by appending the following query param to the URL on the new PR page:
+
+`template=detailed_pr_template.md`
+
+### How do I report a bug?
+
+Found something funky while using gno? You can report it to the team using GitHub issues.
+
+In the issue, please provide as much information as possible, for example:
+
+- what you’ve experienced
+- what should be the expected behavior
+- which version of gno are you running
+- any logs that might be relevant
+
+Once you open up an issue, a core team member will take a look and start a conversation with you about resolving it.
+
+### How do I request a feature?
+
+The gno project operates in the public. This means that feature requests are not just reviewed by the core team, but
+also by the general user base and other contributors.
+
+If you would like to propose a new feature (submit a feature request), you can do so by opening up an issue on GitHub,
+and providing ample context as to why this feature should be implemented.
+
+Additionally, it is encouraged that users suggest their idea on the official communication channels like Discord, to get
+a feel for what other contributors think.
+
+## Style Guides
+
+### Git Commit Messages
+
+The gno project tends to use the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard for any
+commit that goes into the main code stream (currently, the `master` branch).
+
+Each PR is squashed and merged into the main code stream, which means PR _titles_ should also adhere to the Conventional
+Commit standard, and be short and precise.
+
+A general rule of thumb:
+
+- Use Conventional Commits for `master` branch pushes and PR titles
```

Comment:
```suggestion
- Use Conventional Commits PR titles
```

We don't want master direct pushes.

Reply:
Updated the bullet point:

[382fa03](https://github.com/gnolang/gno/pull/469/commits/382fa0331abe8431816064588668bed795572a46)